connectors/http/graphql/element-templates/graphql-outbound-connector.json

```diff
@@ -193,11 +193,15 @@
     "type" : "String"
   }, {
     "id" : "authentication.oauthTokenEndpoint",
-    "label" : "Oauth token endpoint",
+    "label" : "OAuth 2.0 token endpoint",
     "description" : "The OAuth token endpoint",
     "optional" : false,
     "constraints" : {
-      "notEmpty" : true
+      "notEmpty" : true,
+      "pattern" : {
+        "value" : "^(=|http://|https://|secrets|\\{\\{).*$",
```

Comment:
I could use `secretsssss` for instance, or `http://///dldl`, not sure we want a full validation though so feel free to ignore my comment.

Reply:
I guess, we're fine with that. We discussed the same issue with generic URL validation but decided not to go over-strict

Reply:
I will create a ticket to revisit the URL pattern

Reply:
There it is https://github.com/camunda/connectors/issues/2236